index.html

```diff
@@ -139,26 +155,47 @@ <h2 class="tit-abi">Skills</h2>
             <li class="li-abi">Terminal</li>
          </ul>
       </div>         
+      </section>
    </section>
+
    <footer> 
       <section class="contact-us" id="contact-section">
-         <a class="logo" id="logo-footer" href="index.html"> <img  src="resources/image_geometry_1.svg" alt="Mosaic above"> </a>
-         <p class="title" id="contact-tit">Get Started</p>
-         <p class="parrafo" id="descr-contact">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
-            sed do eiusmod tempor incididunt ut labore et dolore magna.
-         </p>
-         <div class="form-container">
-             <form name="contact-form" action="https://formspree.io/f/mzbwzdgd" 
+         <div id="img-left-contact">
+            <a class="logo" id="logo-footer" href="index.html"> <img  src="resources/image_geometry_1.svg" alt="Mosaic above"> </a>     
+         </div>   
+         <div id="form-contact">
+            <p class="title" id="contact-tit">Get started</p>
+            <p class="parrafo" id="descr-contact">
+               Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna
+                aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            </p>
+            <div class="form-container">
+               <form name="contact-form" action="https://formspree.io/f/mzbwzdgd" 
                    method="post" id="contact-form">
-                 <input class="name-form" name="name" type="text" placeholder="Your full name" maxlength="30" required>
-                 <input class="email-form" name="email" type="email" placeholder="Your email address" required>
-                 <textarea class="message-form" name="message" placeholder="Enteryour message here"
-                           maxlength="500" required></textarea>
-                 <div class="message-panel"></div>
-                 <button type="submit" class="button-secc" id="submit-form">Start collaboration</button>
+                  <div id="line1-cf">
+                     <div id="name-group">
+                        <label class="dis-block"  for="name-form">Your name</label>
+                        <input class="dis-block"  id="name-form" name="name" type="text" placeholder="Your full name" maxlength="30" required>
+                     </div>
+                     <div id="email-group">
+                        <label class="dis-block"  for="email-form">Your email</label>
+                        <input class="dis-block"  id="email-form" name="email" type="email" placeholder="Your email address" required>
+                     </div>
+                  </div>   
```

Comment:
- [ ]  Kindly fix the styling of your contact form in both mobile and desktop versions to match the design template
## Your mobile design
![t1](https://user-images.githubusercontent.com/94551484/186992703-16d80909-8094-4996-99b4-ffb5e02f545a.png)
## Your desktop design
![t3](https://user-images.githubusercontent.com/94551484/186992713-e13cdb95-0f58-4163-a7b0-f36cbdcca33e.png)

## Figma template mobile
![p1](https://user-images.githubusercontent.com/94551484/186992764-e063209a-5295-4515-8214-db9f0b4b28dc.png)
## Figma desktop
![p2](https://user-images.githubusercontent.com/94551484/186992777-bcb2caef-3983-428c-99b4-67dc19dd9f67.png)
.